### PR TITLE
job payment when offduty

### PR DESCRIPTION
**Pull request by p4zinee**
cannot receive payment from job when offduty

### CODE REVIEW

**Comment by xenfovn** [comment]
no need extra notification when offduty in loop

**Comment by evanillaa** [comment]
When leaving the service [off duty], the player must receive a salary
	["unemployed"] = {
		label = "Civilian",
		defaultDuty = true,
		grades = {
            ['0'] = {
                name = "Freelancer",
                payment = 10
            },
        },
for example

**Comment by p4zinee** [comment]
Forget about unemployed job just make condition 2 is job.name ~= 'unemployed'

**Comment by p4zinee** [comment]
Offduty in a job kind ambulance mechanic police etc.. and unemployed is two different things

**Comment by evanillaa** [comment]
You want to stop your salary when you leave the service
In all cases, your opinion is correct

**Comment by meistroburger** [comment]
If you're not working, you also not getting paid...

File: server/loops.lua [hunk 1]
```diff
@@ -5,8 +5,12 @@ PaycheckLoop = function()
         local Player = QBCore.Functions.GetPlayer(Players[i])
 
         if Player.PlayerData.job ~= nil and Player.PlayerData.job.payment > 0 then
-            Player.Functions.AddMoney('bank', Player.PlayerData.job.payment)
-            TriggerClientEvent('QBCore:Notify', Players[i], "You received your paycheck of $"..Player.PlayerData.job.payment)
+            if Player.PlayerData.job.onduty then 
+                Player.Functions.AddMoney('bank', Player.PlayerData.job.payment)
+                TriggerClientEvent('QBCore:Notify', Players[i], "You received your paycheck of $"..Player.PlayerData.job.payment)
+            else
+                TriggerClientEvent('QBCore:Notify', Players[i], "You are not on duty and will not receive payment from your job!")
```
Comment (phumberdroz) [inline server/loops.lua:12]: I am not sure if a notification is warranted for not receiving the paycheck.

**Pull request closed without merging**